emotion.d.ts

```diff
@@ -0,0 +1,19 @@
+import '@emotion/react';
+import { SerializedStyles } from '@emotion/react';
+
+declare module '@emotion/react' {
+  export interface Theme {
+    color: {
+      primary: string;
+      positive: string;
+      negative: string;
+    };
+  }
+}
+
+// Prevents errors with using 'CSS' prop on React components
+declare module 'react' {
```

Comment:
I wonder if this is avoidable by changing tsconfig's `"jsx": "react-jsx"` ? That's what the emotion docs seem to indicate.

Reply:
<img width="715" alt="image" src="https://user-images.githubusercontent.com/9061026/208298198-f05d438f-bc26-473a-885c-7749c984bbf6.png">
I did try setting to `react-jsx` initally based on the Emotion docs, but Next.js automatically changes it back to `preserve`

Reply:
Created ticket #42 for this.